src/cascadia/TerminalApp/Tab.cpp

```diff
@@ -6,55 +6,76 @@
 
 using namespace winrt::Windows::UI::Xaml;
 using namespace winrt::Windows::UI::Core;
+using namespace winrt::Microsoft::Terminal::Settings;
+using namespace winrt::Microsoft::Terminal::TerminalControl;
 
-Tab::Tab(GUID profile, winrt::Microsoft::Terminal::TerminalControl::TermControl control) :
-    _control{ control },
-    _focused{ false },
-    _profile{ profile },
-    _tabViewItem{ nullptr }
-{
-    _MakeTabViewItem();
-}
+static const int TabViewFontSize = 12;
 
-Tab::~Tab()
+Tab::Tab(const GUID& profile, const TermControl& control)
 {
-    // When we're destructed, winrt will automatically decrement the refcount
-    // of our terminalcontrol.
-    // Assuming that refcount hits 0, it'll destruct it on its own, including
-    //      calling Close on the terminal and connection.
+    _rootPane = std::make_shared<Pane>(profile, control, true);
+
+    _rootPane->Closed([=]() {
+        _closedHandlers();
+    });
+
+    _MakeTabViewItem();
 }
 
 void Tab::_MakeTabViewItem()
 {
     _tabViewItem = ::winrt::Microsoft::UI::Xaml::Controls::TabViewItem{};
-    const auto title = _control.Title();
-
-    _tabViewItem.Header(title);
+    _tabViewItem.FontSize(TabViewFontSize);
+}
 
-    _control.TitleChanged([=](auto newTitle){
-        _tabViewItem.Dispatcher().RunAsync(CoreDispatcherPriority::Normal, [=](){
-            _tabViewItem.Header(newTitle);
-        });
-    });
+UIElement Tab::GetRootElement()
+{
+    return _rootPane->GetRootElement();
 }
 
-winrt::Microsoft::Terminal::TerminalControl::TermControl Tab::GetTerminalControl()
+// Method Description:
+// - Returns nullptr if no children of this tab were the last control to be
+//   focused, or the TermControl that _was_ the last control to be focused (if
+//   there was one).
+// - This control might not currently be focused, if the tab itself is not
+//   currently focused.
+// Arguments:
+// - <none>
+// Return Value:

```

Comment:
Remove unnecessary comments

Reply:
I'm going to just leave these till the discussion in #890 is done - It'll be easy enough to make one PR to remove them all at once